project-base/templates/Front/Content/Error/error404.html.twig

```diff
@@ -0,0 +1,42 @@
+{% extends 'Front/Layout/layoutWithoutPanel.html.twig' %}
+
+{% block title %}
+    {{ 'Error occurred'|trans }}: {{ status_text }}
+{% endblock %}
+
+{% block content %}
+    <div class="web__line">
+        <div class="web__container">
+            <div class="row">
+                {{ render(controller('App\\Controller\\Front\\BreadcrumbController:indexAction', { routeName: getRoute(), routeParameters: getRouteParams() } )) }}
+
+                <div class="box-error-page">
```

Comment:
I like that you split the templates 👍 

I'm thinking... what do you say on common error page template parent with block error_page_content and in those `error404.html.twig`, `error410.html.twig`, and `error500.html.twig`, only this block would be changed? 

It may help keep the design unified on a specific project and when needed, it can still be separated.

Reply:
c5cfe2a1bbc14adca967a3423758e571e2211bba